lib/symbols.ts

```diff
@@ -234,6 +234,17 @@ export function getSymbolicator(): CSSymbolicator {
     return symbolicator;
 }
 
+export function findProtocolNameInConformanceDescriptor(conformance: string): string | null {
+    const regex = /protocol conformance descriptor for \S+ : \S+\.(\S+) in \S+/g;
+    const matches = regex.exec(conformance);
+
+    if (matches == null || matches.length != 2) {
```

Comment:
- Equality checks should be strict.
- I guess there's no need to check the length, since there's only one group?